src/mixins/date-mixin.ts

```diff
@@ -44,38 +44,38 @@ export default class DateMixin extends Vue {
    * @param date the datetime string in UTC timezone
    * @returns the datetime string in local timezone
    */
-  convertUtcTimeToLocalTime (datetime: string): string {
-    if (!datetime) return null // safety check
+  convertUtcTimeToLocalTime (dateString: string): string {
+    if (!dateString) return null // safety check
 
     const options = {
       year: 'numeric',
       month: '2-digit',
       day: '2-digit',
       hour: '2-digit',
       minute: '2-digit',
-      second: '2-digit',
       hour12: true,
       timeZone: 'America/Vancouver'
     }
 
     // chop off the milliseconds and append "Zulu" timezone abbreviation
     // eg, 2020-08-28T21:53:58Z
-    datetime = datetime.slice(0, 19) + 'Z'
+    dateString = dateString.slice(0, 19) + 'Z'
+    const date = new Date(dateString)
 
     // locale 'en-CA' is the only one consistent between IE11 and other browsers
-    // example output: "2019-12-31 04:00:00 PM"
-    let datetimeLocal = new Intl.DateTimeFormat('en-CA', options).format(new Date(datetime))
+    // example output: "2019-12-31 04:00 PM"
+    let datetimeLocal = new Intl.DateTimeFormat('en-CA', options).format(date)
 
     // misc cleanup
-    datetimeLocal = datetimeLocal.replace(',', '')
+    datetimeLocal = datetimeLocal.replace(', ', ' at ')
```

Comment:
Any thoughts on whether we should make this change to Filings UI?

Reply:
I've added a question for Scott in the ticket.

Reply:
Interesting! Yes, definitely a Scott question. Good for readability tho i think